src/planactions.ts

```diff
@@ -42,3 +43,14 @@ export const submitPlan: ActionCreator<ThunkAction<Promise<AnyAction>, IPlanSubm
         });
     };
 };
+
+export const clearQueue: ActionCreator<ThunkAction<Promise<AnyAction>, null, null, any>> = () => {
+    return async (dispatch: Dispatch) => {
+        dispatch(loading());
+        const queueState = await clearQueueAPI();
+        return dispatch({
+            queueState,
+            type: PlanActionTypes.GETPLANLIST
```

Comment:
This works, but you are reusing `GETPLANLIST` when I think you should be adding a new type for CLEARQUEUE here.

Reply:
Thanks for the suggestion! I added a new enum item. Should there be a corresponding interface as well?

Reply:
You don't need a new interface if there is no response data to return.

Reply:
OK, makes sense. I was thinking about displaying an empty queue on the `Acquire` page. How would I replace the text currently being printed there? (maybe a separate PR)

Reply:
We should create a queue display component if we want to display the queue in multiple pages, definitely a separate PR.